test/toctree.rst

```diff
@@ -1,7 +1,18 @@
 **************************
 Testing for the BookServer
 **************************
-To run the tests, execute ``poetry run pytest`` from the parent of this subdirectory; running them here causes code coverage failures.
+To run the tests, execute ``poetry run pytest`` from the parent of this subdirectory; running them here causes code coverage failures. There are several important command-line options:
+
+--skipdbinit    Skip initialization of the test database. This makes the tests start much faster, at the risk of a corrupt database causing spurious test failures.
+--server_debug  Enable server debug mode. This runs the server in a separate terminal/console, which allow you to set breakpoints, stop the code, etc.
+--log_cli_level LEVEL   Set the `pytest logging level <https://docs.pytest.org/en/6.2.x/logging.html#live-logs>`_. This level affects not just pytest, but the server and all tools run by the tests. Use ``--log_cli_level=INFO`` to provide complete output from the server; the `default logging level <default logging level>` is ``WARNING``.
```

Comment:
I think DEBUG is a lower level than INFO

Reply:
True, but then asyncpg (as I remember) spews incredible quantities of messages. There's probably a way to add a filter to only show messages from the bookserver, but I haven't look at that.